man/mkosi.1

```diff
@@ -1109,6 +1109,12 @@ When used with the \f[C]qemu\f[R] verb, this options sets
 guest\[cq]s RAM.
 Defaults to \f[C]1G\f[R].
 .TP
+\f[B]\f[CB]NspawnKeepUnit=\f[B]\f[R], \f[B]\f[CB]--nspawn-keep-unit\f[B]\f[R]
+When used, this option instructs underlying calls of systemd-nspawn to
+use the current unit scope, instead of creating a dedicated transcient
+scope unit for the containers. This option should be used when mkosi is
+run by a service unit.
+.TP
```

Comment:
You don't need to write raw roff manually, edit `mkosi.md`, the man page is generated from it.

Reply:
Thank you @behrmann, this is fixed in latest push.